tests/Integration/Behaviour/Features/Context/Domain/Product/UpdateOptionsFeatureContext.php

```diff
@@ -29,17 +29,19 @@
 namespace Tests\Integration\Behaviour\Features\Context\Domain\Product;
 
 use Behat\Gherkin\Node\TableNode;
-use PHPUnit\Framework\Assert;
 use PrestaShop\PrestaShop\Core\Domain\Manufacturer\Exception\ManufacturerException;
-use PrestaShop\PrestaShop\Core\Domain\Manufacturer\ValueObject\NoManufacturerId;
 use PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductOptionsCommand;
 use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductException;
-use PrestaShop\PrestaShop\Core\Domain\Product\QueryResult\ProductOptions;
 use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopConstraint;
-use Symfony\Component\PropertyAccess\PropertyAccess;
+use Tests\Integration\Behaviour\Features\Context\Domain\Product\UpdateProduct\AbstractUpdateOptionsFeatureContext;
```

Comment:
```suggestion
use Tests\Integration\Behaviour\Features\Context\Domain\Product\AbstractUpdateOptionsFeatureContext;
```

Reply:
Since the abstracts are common to both suites and context it would make sense that they are in the common namespace

Creating a dedicated `UpdateProduct` sub-namespace makes sense while we are "migrating" all the commands, but in the end, I think all the intermediate contexts and abstract classes should probably be merged into a common single `UpdateProductContext` in the final PR which clears the old ones